src/ensembl/production/metadata/service.py

```diff
@@ -594,6 +666,18 @@ def create_release(data=None):
     return release
 
 
+def create_datasets(data=None):
```

Comment:
Is it actually creating a dataset entry? if not the method name is a bit misleading.

Reply:
Yes, this function returns the Datasets